api/container/containerv2/workers.go

```diff
@@ -55,12 +55,45 @@ type ReplaceWorker struct {
 	WorkerID        string `json:"workerID"`
 }
 
+type VoulemeAttachments struct {
+	VolumeAttachments []VoulemeAttachment `json:"volume_attachments"`
+}
+
+type VoulemeAttachment struct {
+	Id         string     `json:"id"`
+	VolumeInfo Volume     `json:"volume"`
+	Device     DeviceInfo `json:"device"`
+	Name       string     `json:"name"`
+	Status     string     `json:"status"`
+	Type       string     `json:"type"`
+}
+
+type Volume struct {
+	Name string `json:"name"`
+	Id   string `json:"id"`
+}
+
+type DeviceInfo struct {
+	Id string `json:"id"`
+}
+
+type AttachVolume struct {
+	Cluster            string `json:"cluster"`
+	VolumeAttachmentID string `json:"volumeAttachmentID,omitempty"`
+	VolumeID           string `json:"volumeID,omitempty"`
+	Worker             string `json:"worker"`
+}
+
 //Workers ...
 type Workers interface {
 	ListByWorkerPool(clusterIDOrName, workerPoolIDOrName string, showDeleted bool, target ClusterTargetHeader) ([]Worker, error)
 	ListWorkers(clusterIDOrName string, showDeleted bool, target ClusterTargetHeader) ([]Worker, error)
 	Get(clusterIDOrName, workerID string, target ClusterTargetHeader) (Worker, error)
 	ReplaceWokerNode(clusterIDOrName, workerID string, target ClusterTargetHeader) (string, error)
+	GetStorageAttachmentList(clusterIDOrName, workerID string, target ClusterTargetHeader) (VoulemeAttachments, error)
+	GetStorageAttachment(clusterIDOrName, workerID, volumeAttachmentID string, target ClusterTargetHeader) (VoulemeAttachment, error)
+	CreateStorageAttachment(clusterIDOrName, workerID, VolumeID string, target ClusterTargetHeader) (VoulemeAttachment, error)
```

Comment:
CreateStorageAttachment can take the model as input right? 
For delete also we can take model ?